cocos/core/platform/view.ts

```diff
@@ -323,8 +322,10 @@ export class View extends EventTarget {
      * @en Whether to Enable on anti-alias
      * @zh 控制抗锯齿是否开启
      * @param enabled - Enable or not anti-alias
+     * @deprecated since v3.0, view.enableAntiAlias is deprecated now, please use Texture2D.setFilters instead
```

Comment:
Just say it's deprecated, no alternative currently
In the future, we will have MSAA or other method provided in render pipeline